### PR TITLE
Update CSS reset

### PR DESCRIPTION
**Pull request by iamuchejude**

### CODE REVIEW

**Comment by frontibotito** [comment]
Deploy preview for _website_ ready!

Built with commit bfc01a3b572c05ac75b019b7b00a26e3e292e6e7

https://frontity-org-vqitakgs7.now.sh

File: packages/frontity-org-theme/src/components/styles/global-styles.tsx [hunk 1]
```diff
@@ -3,12 +3,24 @@ import FrontityOrg from "../../../types";
...
-    padding: 0;
+  *,
+  *::before,
+  *::after {
+    box-sizing: border-box;
   }
+
+  html {
+    line-height: 1.15;
+    -webkit-text-size-adjust: 100%;
```
Comment (michalczaplinski) [inline packages/frontity-org-theme/src/components/styles/global-styles.tsx]: why do we need that? :)

File: packages/frontity-org-theme/src/components/styles/global-styles.tsx [hunk 2]
```diff
@@ -17,44 +29,164 @@ const cssReset = css`
...
+    border-style: none;
+    padding: 0;
+  }
+
+  button:-moz-focusring,
+  [type="button"]:-moz-focusring,
+  [type="reset"]:-moz-focusring,
+  [type="submit"]:-moz-focusring {
+    outline: 1px dotted ButtonText;
+  }
```
Comment (michalczaplinski) [inline packages/frontity-org-theme/src/components/styles/global-styles.tsx]: what is that? why do we want a dotted outline on the button? 🙂

File: packages/frontity-org-theme/src/components/styles/global-styles.tsx (hunk 1 above)
Comment (iamuchejude) [reply packages/frontity-org-theme/src/components/styles/global-styles.tsx]: The property prevents adjustments of font size after a change in orientation in iOS.

Comment (luisherranz) [reply packages/frontity-org-theme/src/components/styles/global-styles.tsx]: Doesn't the "auto vendor-prefixing" feature of Emotion means that we don't need to use prefixes like `-webkit`?

Comment (iamuchejude) [reply packages/frontity-org-theme/src/components/styles/global-styles.tsx]: Yeah! I didn't know about that. I'm gonna fix it.

**Comment by michalczaplinski** [comment]
@iamuchejude Heey! 👋 Can we merge this?

File: packages/frontity-org-theme/src/components/styles/global-styles.tsx (hunk 2 above)
Comment (iamuchejude) [reply packages/frontity-org-theme/src/components/styles/global-styles.tsx]: This restores the focus outline styles unset by the previous rule.

**Comment by iamuchejude** [comment]
I need to update a part of the code. I'll merge once it's done.

File: packages/frontity-org-theme/src/components/styles/global-styles.tsx (hunk 2 above)
Comment (iamuchejude) [reply packages/frontity-org-theme/src/components/styles/global-styles.tsx]: `ButtonText` is a `System Variable Color`. 

http://www.iangraham.org/books/xhtml1/appd/update-23feb2000.html

**Pull request merged**